src/homepage/components/kaiUiComponents/IconListItem/IconListItem.scss

```diff
@@ -0,0 +1,222 @@
+@import '../../../../kai-ui/src/theme/fonts.scss';
+@import '../../../../kai-ui/src/theme/theme.scss';
+@import '../../../../kai-ui/src/theme/layout.scss';
+@import '../../../../kai-ui/src/kai-font/style.scss';
+
+:root {
+  --il-text-focused: var(--color-gs00);
+  --il-icon-unfocused: var(--color-zeta);
+}
+.kali-list-view{
+  background-color: grey;
+}
+.kai-il {
+  width: auto;
+  min-height: var(--list-item-height);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  overflow: hidden;
+  outline: none;
+  //external css
+ // background-color: red;
+  width: 221px;
+  height: 72px;
+  margin-left: 10px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  top: 61px;
+  padding:0px;
+
+background: #0078D4;
+border-radius: 6px;
+  &:focus {
+    color: var(--il-text-focused);
+  }
+
+  &-icon {
+     &-focused {
+      width: 68px;
```

Comment:
Can we avoid magic numbers like 68px, use rem, and/or move them to variables for reuse